File: pyg_1_3/pyg_parent/pyg_user_web/src/main/java/com/pyg/user/controller/LoginController.java
package com.pyg.user.controller;

import org.springframework.security.core.context.SecurityContextHolder;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RestController;

import java.util.HashMap;
import java.util.Map;

/**
 * 从security中获取用户登陆信息
 */
@RestController
@RequestMapping("login")
public class LoginController {

    @RequestMapping("findLoginUser")
    public Map loginName(){
        Map map = new HashMap();
        String username = SecurityContextHolder.getContext().getAuthentication().getName();
        map.put("loginName", username);
        return map;
    }
}

File: pyg_1_3/pyg_parent/pyg_user_web/src/main/webapp/js/controller/loginController.js
 //控制层 
app.controller('loginController' ,function($scope,loginService){
    // 获取登录用户信息
	$scope.findLoginUser = function () {
        loginService.findLoginUser().success(
			function (response) {
			$scope.loginName = response.loginName;
        })
    };

});
